src/library/logging.hpp

```diff
@@ -48,6 +48,8 @@
 #define LOG_REGION_SOCKET "socket"
 #define LOG_REGION_THCI "thci"
 #define LOG_REGION_TOKEN_MANAGER "token-manager"
+#define LOG_REGION_JOB_MANAGER "job-manager"
+#define LOG_REGION_JOB "job"
```

Comment:
I don't think there is any `job` or `job manager` code in the library. If we want to use the log utilities for the CLI APP, we should move `logging.hpp` to `src/common`.

Reply:
Fixed by moving `logging.{c,h}pp` to `src/common`